boards/arm/mm_swiftio/doc/index.rst

```diff
@@ -0,0 +1,230 @@
+| P40       | GPIO_B1_9     | D40      | GPIO2_IO25    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P41       | GPIO_B1_8     | D41      | GPIO2_IO24    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P42       | GPIO_B1_7     | D42      | GPIO2_IO23    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P43       | GPIO_B1_6     | D43      | GPIO2_IO22    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P44       | GPIO_B1_5     | D44      | GPIO2_IO21    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+| P45       | GPIO_B1_4     | D45      | GPIO2_IO20    |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+|           | GPIO_AD_B1_07 |          |               |          |               |
++-----------+---------------+----------+---------------+ I2C0     | I2C_3         |
+|           | GPIO_AD_B1_06 |          |               |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+|           | GPIO_AD_B1_00 |          |               |          |               |
++-----------+---------------+----------+---------------+ I2C1     | I2C_1         |
+|           | GPIO_AD_B1_01 |          |               |          |               |
++-----------+---------------+----------+---------------+----------+---------------+
+
+
+Programming and Flash
+*************************
+
+Build  applications as usual (see :ref:`build_an_application` for more details).
+
+Configuring a Debug Probe
+=========================
+
+This board is configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`,
+however the :ref:`pyocd-debug-host-tools` do not yet support programming the
+external flashes on this board so you must flash the device by "drag-n-drop"
+
+Configuring a Console
+=====================
+
+Regardless of your choice in debug probe, we will use the OpenSDA
+microcontroller as a usb-to-serial adapter for the serial console.
+
+Connect a USB cable from your PC to Serial of SwiftIO.
+
+Use the following settings with your serial terminal of choice (minicom, putty,
+etc.):
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Flashing
+========
+
+Here is an example for the :ref:`hello_world` application.
+
+Connect a USB cable from your PC to "Serial" port of SwiftIO.
+
+.. code-block:: console
+
+    west build -b mm_swiftio samples/hello_world
+    cp build/zephyr/zephyr.bin /media/<user>/SWIFTIODBGR/
+
+On the ubuntu, DAPLink debug probes  appear on the host computer as a USB disk , which be mount to /media/<user>/SWIFTIODBGR/. <user> is your login name
```

Comment:
Change to:
```
On Ubuntu, DAPLink debug probes appear on the host
computer as a USB disk mounted to ``/media/<user>/SWIFTIODBGR/``,
where ``<user>`` is your login name.
```
and put this before the code-block above (rather than after).